source/_integrations/air_quality.markdown

```diff
@@ -10,7 +10,7 @@ ha_codeowners:
 ha_integration_type: entity
 ---
 
-The `air_quality` base platform allows other integrations to process information about air quality and pollution details. It is used by integrations that provide an `air_quality` sensor - you can find those under the `health` [integrations](/integrations/#health).
+The **Air quality** {% term integration %} allows other integrations to process information about air quality and pollution details. It is used by integrations that provide an `air_quality` sensor - you can find those under the `health` [integrations](/integrations/#health).
```

Comment:
It's an entity component and not an integration?

Reply:
Right. But I think in the user documentation, they are all referred to as integrations now, to remove a bit of the technical jargon. There is a reference further down indicating that this is a building block and not a standalone integration.

Reply:
Was mainly as a question. Don't know what it says on the others like `climate` etc. but I tend to agree with you.

Reply:
Good point. Turns out the building block notice isn't there in all cases yet. But that's a different PR.